### DISCUSSION TITLE
The navigation is not for people new to developing

### DISCUSSION THREAD

**Discussion by lololyric** [community/community#67799] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am total newbie and this website has gots to be the most confusing non new user friendly take you in circles over and over die of frustration website on all of the planetary cosmos.after an hour going in circles and adding repositories I hope to learn to use one day end up saying oh gosh forget it and then I go to hugging face and make picture but then I want to save a project and it sends me right back here. What planet do coders come from its English but...? It should change it's name to get some pile of wishful thinking for your bucket then wait for the matradie he will explain but then you stand there and he never comes <Your question or feedback about the GitHub Code Search and Code View products>

**Comment by MustafaAhmed-star** [comment]
t sounds like you've had a frustrating experience , first you should know that you need patience and practice  .
I am gonna talk  about the most important things in GitHub:
1-Creating a Repository
2-Cloning a Repository
3-Creating and Managing Files
4-Making Changes and Committing
5-Pushing Changes
6-Collaborating
